test/validation/org/apache/cassandra/bridges/ctoolbridge/CToolBridge.java

```diff
@@ -242,22 +312,25 @@ public void nodeTool(int node, String command, String arguments)
         return endpoints;
     }
 
-    public void ssTableSplit(int node, String options, String keyspace_path)
+    public void ssTableSplit(String node, String options, String keyspace_path)
     {
         String fullCommand;
         if (options == "")
         {
-            fullCommand = "ctool run " + DEFAULT_CLUSTER_NAME + " " + node + "\'/home/automaton/cassandra*/tools/bin/sstablesplit /home/automaton/cassandra*/data/data/" + keyspace_path + "\'";
+            fullCommand = "/home/automaton/cassandra/tools/bin/sstablesplit /home/automaton/cassandra/data/data/" + keyspace_path;
+            executeRun(fullCommand, node);
         }
         else{
-            fullCommand = "ctool run " + DEFAULT_CLUSTER_NAME + " " + node + " \'/home/automaton/cassandra*/tools/bin/sstablesplit " + options + " /home/automaton/cassandra*/data/data/" + keyspace_path + "\'";
+            fullCommand = "/home/automaton/cassandra/data/data/" + keyspace_path;
```

Comment:
Yeah that was a miss.. I just fixed it. Apparently using our way of C* installation is not putting these apps on the path, I tried that with nodetool "ctool run CVH 0 'nodetool -h localhost info' which didn't work. Therefore, had to hardcode the path